tests/integration/framework/client/etcd.go

```diff
@@ -64,3 +64,16 @@ func (c *EtcdClient) ListAllKeys(ctx context.Context, prefix string) ([]string,
 
 	return r, nil
 }
+
+func (c *EtcdClient) Get(t *testing.T, ctx context.Context, prefix string, opts ...clientv3.OpOption) []string {
+	t.Helper()
+	resp, err := c.client.Get(ctx, prefix, opts...)
```

Comment:
`clientv3.WithPrefix` ? Or should the parameter not be `prefix` ?

Reply:
I added it as a default option